csrc/device_lower/analysis/tma.cpp

```diff
@@ -816,6 +816,7 @@ class DomainMerger {
     for (auto& item : raw_tma_domain) {
       domain_.domain.emplace_back(
           ValGroupAndItsGraph{std::move(std::get<0>(item)), &id_graph});
+      domain_.info.emplace_back();
```

Comment:
This is quite easy to miss. I think we should disallow directly changing `domain` and `info` since they need to be consistent.

Reply:
I agree. Shall I merge this to clear CI then put the change from struct to class in another PR?

Reply:
Let's do that in another PR.

Reply:
See #2967